public/verify-email.html

```
@@ -0,0 +1,184 @@
+            <i class="fas fa-times-circle text-3xl text-red-600 dark:text-red-400"></i>
+          </div>
+          <h2 class="text-xl font-bold text-gray-800 dark:text-gray-200 mb-2">Verification Failed</h2>
+          <p id="error-msg" class="text-gray-600 dark:text-gray-400 text-sm mb-4">This verification link is invalid or has expired.</p>
+
+          <!-- Resend form -->
+          <div class="mt-2 p-4 bg-gray-50 dark:bg-gray-700/40 border border-gray-200 dark:border-gray-600 rounded-xl text-left">
+            <p class="text-xs font-semibold text-gray-700 dark:text-gray-300 mb-2">
+              <i class="fas fa-envelope mr-1 text-teal-500"></i>Resend verification email
+            </p>
+            <form id="resend-form" class="flex gap-2">
+              <input id="resend-email" type="email" placeholder="your@email.com" required
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)